backend/server/tests/test_getCourse.py

```diff
@@ -0,0 +1,28 @@
+import json 
+import requests 
+import random
+
+from server.routers.model import CONDITIONS
+
+PATH="algorithms/exampleUsers.json"
+
+with open(PATH) as f:
+    USERS = json.load(f)
+
+def test_error():
+    x = requests.get('http://127.0.0.1:8000/courses/getCourse/COMP1234')
+    assert x.status_code == 400
+
+def test_get_a_course():
+    x = requests.get('http://127.0.0.1:8000/courses/getCourse/COMP1521')
+
+    assert x.status_code == 200
+    assert x.json()['code'] == "COMP1521"
+    assert x.json()['path_from'].get("COMP1511") != None
```

Comment:
this should be `is not None`

Reply:
there are multiple times where this same thing happens

Reply:
:oo okok I just google the difference of `== None` and `is None`. Will change this.